packages/fluentui/react-northstar/src/components/Datepicker/DatepickerCalendarHeader.tsx

```diff
@@ -0,0 +1,147 @@
+  accessibility?: Accessibility<DatepickerCalendarHeaderBehaviorProps>;
+
+  /** Localized labels */
+  localizedStrings?: IDateGridStrings;
+
+  /** Action to happen on click on the previous button */
+  onPreviousClick?: ComponentEventHandler<DatepickerCalendarHeaderActionProps>;
+
+  /** Action to happen on click on the next button */
+  onNextClick?: ComponentEventHandler<DatepickerCalendarHeaderActionProps>;
+
+  /** Shorthand for the button that navigates to the previous calendar screen. */
+  previousButton?: ShorthandValue<DatepickerCalendarHeaderActionProps>;
+
+  /** Shorthand for the button that navigates to the next calendar screen. */
+  nextButton?: ShorthandValue<DatepickerCalendarHeaderActionProps>;
+}
+
+export type DatepickerCalendarHeaderStylesProps = never;
+
+export const datepickerCalendarHeaderClassName = 'ui-datepicker__calendarheader';
+
+/**
+ * A DatepickerCalendarHeader is used to display header block above calendar grid.
+ * This component is currently UNSTABLE!
+ */
+export const DatepickerCalendarHeader: ComponentWithAs<'div', DatepickerCalendarHeaderProps> &
+  FluentComponentStaticProps<DatepickerCalendarHeaderProps> = props => {
+  const context = useFluentContext();
+  const { setStart, setEnd } = useTelemetry(DatepickerCalendarHeader.displayName, context.telemetry);
+  setStart();
+
+  const {
+    className,
+    design,
+    styles,
+    variables,
+    content,
+    nextButton,
+    previousButton,
+    onPreviousClick,
+    onNextClick,
+  } = props;
+  const ElementType = getElementType(props);
+  const unhandledProps = useUnhandledProps(DatepickerCalendarHeader.handledProps, props);
+  const getA11yProps = useAccessibility(props.accessibility, {
+    debugName: DatepickerCalendarHeader.displayName,
+    actionHandlers: {},
+    rtl: context.rtl,
+  });
+
+  const { classes } = useStyles<DatepickerCalendarHeaderStylesProps>(DatepickerCalendarHeader.displayName, {
+    className: datepickerCalendarHeaderClassName,
+    mapPropsToInlineStyles: () => ({
+      className,
+      design,
+      styles,
+      variables,
+    }),
+    rtl: context.rtl,
+  });
+
+  const element = (
+    <ElementType
+      {...getA11yProps('root', {
+        className: classes.root,
+        ...unhandledProps,
+      })}
+    >
+      <Text content={content} />
+      {createShorthand(DatepickerCalendarHeaderAction, previousButton, {
+        defaultProps: () =>
+          getA11yProps('previousButton', {
+            className: datepickerCalendarHeaderClassName,
+            icon: {},
+            title: 'Previous Month',
+            onClick: onPreviousClick,
+            direction: 'previous',
+          }),
+      })}
+      {createShorthand(DatepickerCalendarHeaderAction, nextButton, {
+        defaultProps: () =>
+          getA11yProps('nextButton', {
+            className: datepickerCalendarHeaderClassName,
+            icon: {},
+            title: 'Previous Month',
```

Comment:
Should this come from `localizedStrings`?

Reply:
Would update it after #14058 extends the localizationStrings interface, adding a comment for now